python/ray/autoscaler/node_provider.py

```diff
@@ -117,6 +117,12 @@ def create_node(self, node_config: Dict[str, Any], tags: Dict[str, str],
         """
         raise NotImplementedError
 
+    def update_node(self, node_ids: List[str]) -> None:
+        """Only for AWS node_provider
+        Check whether update operations are needed in cloudwatch related configs
```

Comment:
I would recommend following the `terminate_node` and `terminate_nodes` pattern below. So, we could change the name of this to `updates_nodes` with a default implementation of:

```python
for node_id in node_ids:            
  logger.info("NodeProvider: {}: Updating node".format(node_id))
  self.update_node(node_id)
```
where `update_node` just has `pass` for its default implementation.

Also, we could change the doc string for each to match the doc string for `terminate_node` and `terminate_nodes`, replacing occurrences of "Terminates" with "Updates".